lib/saml_idp/request.rb

```diff
@@ -67,6 +67,12 @@ def request
       end
     end
 
+    def force_authn
```

Comment:
this looks like a predicate method, WDYT about renaming it?

```suggestion
    def force_authn?
```

Reply:
Good idea, thanks!